src/styles.css

```
@@ -444,10 +444,127 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
+  background: linear-gradient(90deg, var(--orange) 0%, var(--blue) 50%, var(--purple) 100%);
+  z-index: 1;
+}
+
+.premium-footer a {
+  transition: all 0.2s ease-in-out;
+}
+
+.footer-stat-card {
+  background: rgba(255, 255, 255, 0.03);
+  backdrop-filter: blur(8px);
+  -webkit-backdrop-filter: blur(8px);
```

Stylelint (17.12.0)

[error] Vendor-prefixed property "-webkit-backdrop-filter" (property-no-vendor-prefix)

(property-no-vendor-prefix)